pricing.js

```diff
@@ -1,37 +1,49 @@
-module.exports.calculateProductPrice = function (product, employee, coverageLevels) {
+function calculateVolLifePrice(product, coverageLevels) {
```

Comment:
Great job breaking this up into it's own function and the same with the `calculateLTDPrice` function. That's a great first step.